{"cve": {"id": "CVE-2024-50636", "sourceIdentifier": "cve@mitre.org", "published": "2024-11-11T23:15:05.833", "lastModified": "2024-11-12T13:55:21.227", "vulnStatus": "Awaiting Analysis", "cveTags": [], "descriptions": [{"lang": "en", "value": "PyMOL 2.5.0 contains a vulnerability in its \"Run Script\" function, which allows the execution of arbitrary Python code embedded within .PYM files. Attackers can craft a malicious .PYM file containing a Python reverse shell payload and exploit the function to achieve Remote Command Execution (RCE). This vulnerability arises because PyMOL treats .PYM files as Python scripts without properly validating or restricting the commands within the script, enabling attackers to run unauthorized commands in the context of the user running the application."}, {"lang": "es", "value": "PyMOL 2.5.0 contiene una vulnerabilidad en su funci\u00f3n \"Run Script\", que permite la ejecuci\u00f3n de c\u00f3digo Python arbitrario incrustado en archivos .PYM. Los atacantes pueden manipular un archivo .PYM malicioso que contenga un payload de shell inversa de Python y explotar la funci\u00f3n para lograr la ejecuci\u00f3n remota de comandos (RCE). Esta vulnerabilidad surge porque PyMOL trata los archivos .PYM como scripts de Python sin validar o restringir adecuadamente los comandos dentro del script, lo que permite a los atacantes ejecutar comandos no autorizados en el contexto del usuario que ejecuta la aplicaci\u00f3n."}], "metrics": {}, "references": [{"url": "https://github.com/schrodinger/pymol-open-source/issues/405", "source": "cve@mitre.org"}, {"url": "https://github.com/yamerooo123/CVE/blob/main/CVE-2024-50636/Description.md", "source": "cve@mitre.org"}, {"url": "https://youtu.be/SWnN_a1tUNc", "source": "cve@mitre.org"}]}}